opennms-doc/guide-admin/src/asciidoc/text/operation/jmx-config-generator.adoc

```diff
@@ -1,59 +1,252 @@
+=== JMX Config Generator
 
-// Allow GitHub image rendering
-:imagesdir: ../../images
+The _JMX Config Generator_ is a revised and improvised version of the original _JmxConfigTool_.
+It provides a fast and easy way to generate a valid and functional `jmx-datacollection-config.xml` file from a running Java application.
+Based on this generated configuration file it can also generate a `snmp-graph.properties` file with matching graph definitions.
 
-=== JmxConfigTool
-The JmxConfigTool provides a fast and easy way to generate a proper _jmx-datacollection-config_ from a running Java application.
-Based on the generated configuration file it can also provide a set of _snmp-graph_ configurations.
+==== Installation
+From current _OpenNMS_ versions 1.12 and later the _JMX Config Generator_ is included as a separate installable package.
 
-The JmxConfigTool offers two modes:
+===== Installation on Ubuntu
 
-==== Jmx Mode
-The _Jmx mode_ generates an _jmx-datacollection-config_ file to provide all relevant configurations to collect metrics from a Java Application and/or a Java Virtual Machine.
-The network connection to the JVM and the running Java Application is RMI based and can provide credentials.
-The following graphic offers an overview about the _Jmx mode_.
+Install the package from the OpenNMS repository:
+
+[source, shell]
+----
+$ sudo apt-get install opennms-jmx-config-generator
+Reading package lists... Done
+Building dependency tree
+Reading state information... Done
+The following NEW packages will be installed:
+  opennms-jmx-config-generator
+0 upgraded, 1 newly installed, 0 to remove and 6 not upgraded.
+Need to get 2,180 kB of archives.
+After this operation, 2,225 kB of additional disk space will be used.
+Get:1 http://debian.opennms.eu/ stable/main opennms-jmx-config-generator all 16.0.2-1 [2,180 kB]
+Fetched 2,180 kB in 0s (3,275 kB/s)
+Selecting previously unselected package opennms-jmx-config-generator.
+(Reading database ... 153349 files and directories currently installed.)
+Preparing to unpack .../opennms-jmx-config-generator_16.0.2-1_all.deb ...
+Unpacking opennms-jmx-config-generator (16.0.2-1) ...
+Setting up opennms-jmx-config-generator (16.0.2-1) ...
+----
```

Comment:
It's mentioned above: "Install the package from the _OpenNMS_ repository". So, I think this is okay.